changelog/fix-1566-checkout_block_account_suspended

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add exception handling to prevent fatal error
```

Comment:
This changelog is added to the public changelog of WC Pay, so what do you think of including more details about the issue here? IMO something along the lines of "Prevent fatal errors when fetching payment methods on checkout block" would be a bit more informative (feel free to reword it 😄).

Reply:
Similar to the comment above, I think the PR name could use some more details about what error is being fixed.